### PR TITLE
adding print for validation callback

### PR DESCRIPTION
**Pull request by jperez999**
adds print statement for validation callback in tf dataloader.

### CODE REVIEW

**Review by benfred — APPROVED** [review]

File: nvtabular/loader/tensorflow.py [hunk 1]
```diff
@@ -355,6 +355,10 @@ def on_epoch_end(self, epoch, logs={}):
             for metric in self.model.metrics:
                 metric.update_state(y_true, y_pred)
 
+        set_logs = {}
         for metric in self.model.metrics:
-            logs["val_" + metric.name] = metric.result().numpy()
+            name = "val_" + metric.name
```
Comment (benfred) [inline nvtabular/loader/tensorflow.py:360]:
```suggestion
```

**Comment by jperez999** [comment]
rerun tests

**Comment by jperez999** [comment]
rerun tests

**Comment by jperez999** [comment]
rerun tests

**Comment by jperez999** [comment]
testing in CI

**Comment by jperez999** [comment]
rerun tests

**Comment by jperez999** [comment]
rerun tests

**Pull request merged**